app/feature/feature-movingflow/src/main/kotlin/com/hedvig/android/feature/movingflow/ui/addhouseinformation/AddHouseInformationViewModel.kt

```diff
@@ -225,7 +225,8 @@ private fun MovingFlowState.toInputForSubmission(validContent: ValidAddressInput
       ),
       moveFromAddressId = moveFromAddressId,
       movingDate = this.movingDateState.selectedMovingDate!!,
-      numberCoInsured = this.propertyState.numberCoInsuredState.selectedNumberCoInsured,
+      // property state should be set before
+      numberCoInsured = this.propertyState?.numberCoInsuredState?.selectedNumberCoInsured!!,
```

Comment:
Is the `!!` in reality something that should go on the `propertyState!!` instead, if that is what we assert that will not be null here?